.github/workflows/api_docs.yml

```
@@ -23,7 +23,7 @@ jobs:
         id: version_finder
         run: |
           curl -s "https://dash.readme.com/api/v1/version" --header "authorization: Basic ${{ secrets.README_API_KEY }}" > out
```

nit: just a short comment so the next one bumping into this can quickly understand what this long jq command is doing

```suggestion
          curl -s "https://dash.readme.com/api/v1/version" --header "authorization: Basic ${{ secrets.README_API_KEY }}" > out
          # Ensure proper semantic version sorting (2.2.0 comes before 2.10.0)
```